app/objects/c_ability.py

```diff
@@ -93,7 +93,11 @@ def __init__(self, ability_id, tactic=None, technique_id=None, technique=None, n
             self.access = self.Access(access)
         self.additional_info = dict()
         for k, v in kwargs.items():
-            self.additional_info[k] = v
+            if k == 'additional_info':
+                for k2, v2 in v.items():
```

Comment:
this block can be done much cleaner. I consider utilizing:
1) comprehensions to make the code smaller
2) extracting this block into a private function, _apply_additional_info()

Also:
1) ensure that additional_info is not a hard-coded string.  
2) ensure we add this to documentation